README.md

```diff
@@ -72,6 +72,7 @@ The files:
 - `stinit.c`: The stinit source
 - `stinit.8`: The man page for stinit
 - `stinit.def.examples`: example configurations for different devices
+- `mt-st`: bash auto completion file
```

Comment:
I think for clarity of the files in the repo, this would be better named `mt-st.bash_completion`. What do you think?

Reply:
Sure, no problem.